chains/orchestrator-paras/runtime/flashbox/src/lib.rs

```diff
@@ -490,6 +490,7 @@ impl cumulus_pallet_parachain_system::Config for Runtime {
     type CheckAssociatedRelayNumber = RelayNumberMonotonicallyIncreases;
     type ConsensusHook = ConsensusHook;
     type SelectCore = cumulus_pallet_parachain_system::DefaultCoreSelector<Runtime>;
+    type RelayParentOffset = ConstU32<0>;
```

Comment:
what is this?

Reply:
Added to the PR description, its the same feature as `relay_parent_descendants`: https://github.com/paritytech/polkadot-sdk/pull/8299